pom.xml

```diff
@@ -29,18 +30,24 @@
 
   <properties>
     <jave.version>3.0.1</jave.version>
-    <vertx.version>4.0.3</vertx.version>
+    <vertx.version>4.2.0.Beta1</vertx.version>
```

Comment:
Probably just my paranoia, but I always feel a little hinky about using beta releases in dependencies... does 2.0.Beta1 have major updates that make it preferable to 4.1.4?

Reply:
My thinking was this is beta too (until we turn it into a real service), but it's a reasonable concern. I'll bump back down to 4.1.4. Thanks.